Frameworks/UIKit/NSLayoutManager.mm

```diff
@@ -40,189 +40,7 @@ static float invokeWidthBlock(void* opaque, CFIndex idx, float offset, float hei
 }
 
 - (void)_layoutAllText {
```

Comment:
#1132 deals with this, but I needed to get rid of this locally to unbreak some internal test apps.